plugins/pulp_rpm/plugins/distributors/yum/metadata/updateinfo.py

```diff
@@ -85,6 +85,38 @@ def _repo_unit_nevra(self, erratum_unit, repo_id):
 
         return repo_unit_nevra
 
+    def _get_package_checksum_tuple(self, package):
+        """
+        Decide which checksum to publish for the given package in the erratum package list.
+
+        Handle two possible ways of specifying the checksum in the erratum package list:
+        - in the `sum` package field as a list of alternating checksum types and values,
+          e.g. ['type1', 'checksum1', 'type2', 'checksum2']
+        - in the `type` and `sums` package fields. It is only the case when the erratum was uploaded
+          via pulp-admin. Only one type of the checksum could be specified this way.
+
+        :param package: package from the erratum package list
+        :type  package: dict
+        :return: checksum type and value to publish
```

Comment:
This should describe that an empty tuple can be returned, and what circumstance that indicates.